index.html

```diff
@@ -46,26 +45,36 @@ <h1>AutoYang 1000</h1>
             <img src="images/yang-gang-2020-baseball-cap.png" style="width: 100px; height: auto touch-action: none" class="item">
             <img src="images/yang-logo.png" style="width: 100px; height: auto; cursor: pointer; touch-action: none" class="item">
 
+            <div id="output-block" style='display:none'>
+              <h6>Save this image!</h6>
```

Comment:
I would suggest bold font and wording like, Right click or long press to save image.